packages/client/src/views/SysAdmin/Config/Application/Application.tsx

```diff
@@ -358,126 +121,44 @@ function BirthTabContent({
         <>
           {section.title}
           <ListViewSimplified key={idx}>
-            {section.items.map((item, index) => {
-              const id = isString(item.label)
-                ? item.label.split(' ').join('-')
-                : 'label-component'
-              return (
-                <ListViewItemSimplified
-                  key={index}
-                  label={<Label id={`${id}_label`}>{item.label}</Label>}
-                  value={<Value id={`${id}_value`}>{item.value}</Value>}
-                  actions={
-                    <LinkButton
-                      id={item.action.id}
-                      disabled={item.action.disabled}
-                      onClick={item.action.handler}
-                    >
-                      {item.action?.label}
-                    </LinkButton>
-                  }
-                />
-              )
-            })}
+            {idx === 0 && (
```

Comment:
I don't think we need to keep separate sections here and map them to `ListViewSimplified`, instead it would be better if we just write them in tsx directly

Reply:
Agreed. Changed the code.